content/how-to/cloud-servers/using-rsync-to-synchronize-file/index.md

```diff
@@ -0,0 +1,82 @@
+---
+permalink: '/using-rsync-to-synchronize-files'
```

Comment:
I don't think you need the ' before and after the permalink.

Reply:
I have made the correction, Chris, thanks for letting me know.